pkg/audit/verifier.go

```diff
@@ -61,7 +61,7 @@ func NewVerifier(transport transport.Client, overlay overlay.Client, id provider
 func (d *defaultDownloader) getShare(ctx context.Context, stripeIndex, shareSize, pieceNumber int,
 	id psclient.PieceID, pieceSize int64, fromNode *pb.Node, authorization *pb.SignedMessage) (s share, err error) {
 	defer mon.Task()(&ctx)(&err)
-
+	fromNode.Type.PanicOnInvalid("audit getShare")
```

Comment:
i would definitely vote against panics. also, in what scenario would a node even have an invalid type? is INVALID just if a node type is unset?

Reply:
The panics are in place to find all the places where we haven't properly set the NodeType. They will be removed before final merge.

Yes, Invalid is to ensure that we don't leave node type empty.

Reply:
oh nice i just came back to reply to my own comment after i saw the todo in PanicOnInvalid. good work, carry on